recipes/jupyter-dashboards/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "cb198c9e85e259dbdd34c3dbdaf97e9c63fa6c7ee86caa5bbf719859a799f3db" %}
+{% set name = "jupyter_dashboards" %}
+
+package:
+  name: {{ name }} 
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - {{ name.replace('_', '-') }} = jupyter_dashboards.extensionapp:main 
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - notebook
+    - python
+    - traitlets
+
+test:
+  imports:
+    - {{ name }} 
+  commands:
+    - {{ name.replace('_', '-') }} --help
```

Comment:
Ok with the jinja if there will be many similar recipes to this one. If not, maybe we can drop it.